pybaselines/_algorithm_setup.py

```diff
@@ -21,8 +23,8 @@ def difference_matrix(data_size, diff_order=2):
     ----------
     data_size : int
         The number of data points.
-    diff_order : {2, 0, 1, 3, 4, 5}, optional
-        The integer differential order; either 0, 1, 2, 3, 4, or 5. Default is 2.
+    diff_order : int, optional
+        The integer differential order; must be positive. Default is 2.
```

Comment:
say "must be >= 0" instead, since positive is a little ambiguous since 0 is allowed